verification/src/compiler/compilers.rs

```diff
@@ -1,26 +1,26 @@
-use crate::compiler::{CompilerVersion, DownloadCache, Fetcher, VersionList};
-use anyhow::anyhow;
+use super::fetcher::FetchError;
+use crate::compiler::{self, DownloadCache, Fetcher};
 use ethers_solc::{artifacts::Severity, error::SolcError, CompilerInput, CompilerOutput, Solc};
-use std::fmt::{Debug, Display};
-use thiserror::Error;
+use std::{fmt::Debug, sync::Arc};
+use thiserror::Error as DeriveError;
 
-#[derive(Debug, Error)]
-pub enum CompilersError {
+#[derive(Debug, DeriveError)]
+pub enum Error {
     #[error("Error while fetching compiler: {0:#}")]
-    Fetch(anyhow::Error),
+    Fetch(FetchError),
```

Comment:
```suggestion
    Fetch(#[from] FetchError),
```
Allows to get rid of `map_err` and use only `?`

Reply:
why we're not using it everywhere then? :thinking:

Reply:
good question :) 
I didn't write it somewhere because if you had 2 similar #[from] errors it wouldn't work. 
but yes, we should use it when it's possible